server/src/test/java/org/candlepin/TestingModules.java

```diff
@@ -148,6 +173,39 @@ private TestingModules() {
         // This class is just a container for various Guice Modules used during testing
     }
 
+    public static ModelTranslator buildTranslator() {
+        ModelTranslator modelTranslator = new SimpleModelTranslator();
+
+        // API translators
+        /////////////////////////////////////////////
+        modelTranslator.registerTranslator(
+            new CertificateSerialTranslator(), CertificateSerial.class, CertificateSerialDTO.class);
+        modelTranslator.registerTranslator(
+            new CertificateTranslator(), Certificate.class, CertificateDTO.class);
+        modelTranslator.registerTranslator(
+            new ConsumerTypeTranslator(), ConsumerType.class, ConsumerTypeDTO.class);
+        modelTranslator.registerTranslator(
+            new ContentTranslator(), Content.class, ContentDTO.class);
+        modelTranslator.registerTranslator(
+            new OwnerTranslator(), Owner.class, OwnerDTO.class);
+        modelTranslator.registerTranslator(
+            new ProductTranslator(), Product.class, ProductDTO.class);
+        modelTranslator.registerTranslator(
+            new UpstreamConsumerTranslator(), UpstreamConsumer.class, UpstreamConsumerDTO.class);
+
+        // Shims
+        /////////////////////////////////////////////
+        modelTranslator.registerTranslator(
+            new ContentDataTranslator(), ContentData.class, ContentDTO.class);
+        modelTranslator.registerTranslator(
+            new ProductDataTranslator(), ProductData.class, ProductDTO.class);
+
+
+        return modelTranslator;
```

Comment:
The setting up of the translator for testing seems to be duplicated with that of the candlepin module. Could we instead create a new DefaultModelTranslator (or the likes) and have both guice modules instantiate the instance? At least this way we don't have to remember to register the translators in multiple places. This would still allow the tests flexibility if it needed to special case anything.

Reply:
Added StandardTranslator.